demosplan/DemosPlanCoreBundle/Entity/Statement/Statement.php

```diff
@@ -4111,4 +4195,9 @@ public function setAnonymous(bool $anonymous): Statement
 
         return $this;
     }
+
+    public function setSimilarStatementSubmitter(ProcedurePerson $similarStatementSubmitter): void
+    {
+        $this->similarStatementSubmitters = new ArrayCollection([$similarStatementSubmitter]);
+    }
```

Comment:
This does not add this `Statement` to the `ProcedurePerson` nor does it remove this `Statement` from the `ProcedurePersons` that were present in the old `Collection`.

Reply:
I've missed that one, thanks.
8a1acd82d1e2dc2b747db09512511fae09198f46